File: backend/test_conn.py
import mysql.connector
import sys

print("Testing connection with Python...")
try:
    conn = mysql.connector.connect(
        host="bsyfwkfz5d6scbiu2ue8-mysql.services.clever-cloud.com",
        user="uqpbzumyuo5nhmvw",
        password="3f8GkeQfYhNtDuql63eU",
        database="bsyfwkfz5d6scbiu2ue8",
        port=3306
    )
    print("✅ Connection SUCCESS with Python!")
    conn.close()
except Exception as e:
    print("❌ Connection FAILED with Python:", e)
    sys.exit(1)

File: backend/test_conn.js
const mysql = require('mysql2/promise');

async function test() {
  console.log('Testing connection with Connection URI...');
  try {
    const conn = await mysql.createConnection('mysql://uqpbzumyuo5nhmvw:3f8GkeQfYhNtDuql63eU@bsyfwkfz5d6scbiu2ue8-mysql.services.clever-cloud.com/bsyfwkfz5d6scbiu2ue8');
    console.log('✅ Connection SUCCESS with URI!');
    await conn.end();
  } catch (err) {
    console.error('❌ Connection FAILED with URI:', err);
  }
}

test();
